docs/quick-and-dirty-ocaml414.md

```diff
@@ -0,0 +1,55 @@
+These instructions assume you want a local opam switch.
+If you don't want that, replace "." with the name of your switch.
+
+From project root folder:
+
+1. Replace the current opam switch with a fresh one for OCaml 4.14.0.
+
+``` shell
+opam switch remove .
+opam switch create . 4.14.0
+```
+
+2. Make sure this switch remains selected when you leave the project folder.
+
+``` shell
+eval $(opam env --switch=. --set-switch)
+```
+
+3. From somewhere outside the Hazel source tree, clone the following opam package repos:
+
+``` shell
+cd ..
+git clone 'git@github.com:janestreet/async_js.git'
+git clone 'git+https://github.com/janestreet/incr_dom.git'
```

Comment:
git+https doesn't parse on my machine

Reply:
> git+https doesn't parse on my machine

Not sure what you mean. What did you try? I gave `opam pin` some https URLs and it converted them into what you see there.